internal/store/postgres/servicedata.go

```diff
@@ -20,16 +21,26 @@ type Key struct {
 }
 
 type ServiceData struct {
-	URN         string `db:"urn"`
-	NamespaceID string `db:"namespace_id"`
-	EntityID    string `db:"entity_id"`
-	Value       string `db:"value"`
-	Key         string `db:"key"`
-	ProjectID   string `db:"project_id"`
-	ResourceID  string `db:"resource_id"`
+	URN         string         `db:"urn"`
+	NamespaceID string         `db:"namespace_id"`
+	EntityID    string         `db:"entity_id"`
+	Value       sql.NullString `db:"value"`
```

Comment:
Should this be `map[string]any` ?

Probably we can use same type like this https://github.com/goto/siren/blob/main/pkg/pgc/type.go#L11

Reply:
No mas, Value can be both, json or a string.

Reply:
Why it could be string? The column is in jsonb right? 
Lets stick and enforce with a single type of data and not mixing this up?

Reply:
We already support both kind of metadata

`
    "metadata": {
        "kx": "string type value"
    }
`
and
`
    "metadata": {
        "kx": {
            "k1": "v1"
        }
    }
`

Reply:
Yeah I know but that just wrong to support both. Either we stick with string or json.

we should be consistent in service data one